lib/inspec/resources/oracle_db_conf.rb

```diff
@@ -0,0 +1,42 @@
+# copyright: 2015, Vulcano Security GmbH
```

Comment:
No need for this copyright line - this is from an acquired company that originated InSpec.